templates/registration/login.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{%  block content %}
+<h1>Login</h1>
+  {% if form.errors %}
+  <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form action="{% url 'login' %}" method="post">
+  {% csrf_token %}
+    {{ form.as_p }}
+    <input type="submit" value="Submit">
+    <input type="hidden" name="next" value="{{ next }}">
```

Comment:
The `next` input is used to redirect users after login. Ensure that the `next` context variable is correctly set in your view to handle redirection after successful login.